Mathlib/RingTheory/Finiteness.lean

```diff
@@ -306,7 +314,7 @@ theorem fg_of_fg_map_of_fg_inf_ker {R M P : Type _} [Ring R] [AddCommGroup M] [M
       rw [ht2] at this
       exact this.1
   intro x hx
-  have : f x ∈ map f s := by
+  have : asFun f x ∈ @map R R M P _ _ _ _ _ _ (RingHom.id _) _ _ instSLMC f s := by
```

Comment:
With `etaExperiment`, we don't need the `asFun` and `@` in the following statement.

Reply:
```suggestion
  -- Porting note: With etaExperiment, we don't need the asFun and @ in the following statement.
  have : asFun f x ∈ @map R R M P _ _ _ _ _ _ (RingHom.id _) _ _ instSLMC f s := by
```